app/controllers/contestants_controller.rb

```diff
@@ -0,0 +1,10 @@
+class ContestantsController < ApplicationController
+  def index
+    @contestants = Contestant.all
+    @projects = Project.all
+    @challenges = Challenge.all
+    @contestant_projects = ContestantProject.all
+
+    # binding.pry
```

Comment:
For the contestants index, you only need ALL of the contestants. Lines 4-6 are unnecessary. Also, something to note, is that rails convention is to only send one object to the view.